src/engine/strat_engine/mdv.rs

```diff
@@ -73,7 +79,9 @@ impl<'a> MountedMDV<'a> {
 impl<'a> Drop for MountedMDV<'a> {
     fn drop(&mut self) {
         if let Err(err) = umount(&self.mdv.mount_pt) {
-            warn!("Could not unmount MDV: {}", err)
+            warn!("Could not unmount MDV: {}", err);
```

Comment:
The drop method needs a comment explaining why this is happening. Not a header comment, just a regular one.

Reply:
you mean, why we're trying to umount, or why the umount might have failed?

Reply:
in any case, broke out this commit into its own PR, #689, so let's continue over there.